test/Microsoft.AspNet.Mvc.Core.Test/ActionResults/PartialViewResultTest.cs

```diff
@@ -167,6 +168,13 @@ public class PartialViewResultTest
                            .Returns(viewEngine.Object);
             serviceProvider.Setup(p => p.GetService(typeof(ILogger<PartialViewResult>)))
                            .Returns(new Mock<ILogger<PartialViewResult>>().Object);
+            serviceProvider.Setup(s => s.GetService(typeof(IOptions<MvcOptions>)))
+                .Returns(() => {
```

Comment:
Not for this PR: In the future please _never_ use trailing curly braces, even in test code.  Curly braces get their own lines except in the special case of single line auto-properties and sometimes close parentheses.  e.g.
``` c#
    serviceProvider.Setup(s => s.GetService(typeof(IOptions<MvcOptions>)))
        .Returns(() =>
        {
            var optionsAccessor = new Mock<IOptions<MvcOptions>>();
            ...
        });
```